website/maps.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1"
+    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="stylesheet" href="../src/bottomsheet.css" />
+  </head>
+
+  <body>
+    <!-- <div class="main-div">
+        <button id="target-1" data-bottomsheet-id="maps-1">click</button>
+    </div> -->
+
+    <div id="maps-2" data-bottomsheet>
+      <div class="flex border">
+        <p class="sm">Set Up Contact Card</p>
+        <p class="more" id="done">Done</p>
+      </div>
+      <div class="flex center">
+        <figure class="round">
+          <img src="./assets/home.svg" width="30" height="30" />
+        </figure>
+        <figure class="round margin">
+          <img src="./assets/work.svg" width="30" height="30" />
+        </figure>
+        <figure class="round margin">
+          <img src="./assets/pin.svg" width="30" height="30" />
+        </figure>
+      </div>
+      <div>
+        <p class="md text-center">
+          Maps uses the home, work and school adresses from your Contact card
+        </p>
+        <p class="fav text-center">
+          Keep your Contact Card up to date to see your Home, Work and School in
+          Maps
+        </p>
+      </div>
+      <button class="contact" id="target-map-3" data-bottomsheet-id="maps-3">
+        Open My Contact Card
+      </button>
+    </div>
+    <div id="maps-3" data-bottomsheet>
+      <div class="cont-box no-flex text-center">
+        <p class="fav border p2">No Existing Contact Card</p>
+        <p class="lg more border">Create a new one</p>
+        <p class="lg more">Choose an existing one</p>
+      </div>
+
+      <button class="cancel more lg">Cancel</button>
+    </div>
+    <script type="module" src="./maps.js"></script>
```

Comment:
```suggestion
    <script type="module" src="/maps.js"></script>
```

Having `.` also will not resolve correctly in vite